### PR TITLE
Solves bug in #18482

### PR DESCRIPTION
**Pull request by Saanidhyavats**
#### Brief description of what is fixed or changed
This request fixes the bug related to issue #18482 

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Saanidhyavats** [comment]
Some commits which is not related to #18482 will be removed

File: sympy/series/tests/test_gruntz.py [hunk 1]
```diff
@@ -467,3 +467,8 @@ def test_issue_6682():
 def test_issue_7096():
     from sympy.functions import sign
     assert gruntz(x**-pi, x, 0, dir='-') == oo*sign((-1)**(-pi))
+
+
+def test_issue_18482():
+    x = Symbol('x',positive=True,real=True)
```
Comment (Smit-create) [inline sympy/series/tests/test_gruntz.py]: Add spaces after commas.

File: sympy/series/gruntz.py [hunk 2]
```diff
@@ -626,9 +626,6 @@ def rewrite(e, Omega, x, wsym):
 
     # Some parts of sympy have difficulty computing series expansions with
     # non-integral exponents. The following heuristic improves the situation:
-    exponent = reduce(ilcm, denominators, 1)
```
Comment (Smit-create) [inline sympy/series/gruntz.py]: As you have removed this line, you need to remove `ilcm` from sympy import at the top of the file.

**Comment by Smit-create** [comment]
Please make the title more descriptive(short and precise), and related to the change you have made.

**Review by Arpan612 — COMMENTED** [review]
Why don't you let the import stay in gruntz.py?....It won't cause any error imo.

**Comment by Smit-create** [comment]
@Arpan612  It will lead to buil failure because of flake8 error.

**Comment by smichr** [comment]
please squash all the commits into one and repush here with the `-f` option

File: sympy/series/gruntz.py [hunk 3]
```diff
@@ -626,9 +625,6 @@ def rewrite(e, Omega, x, wsym):
...
```
Comment (Sc0rpi0n101) [inline sympy/series/gruntz.py]: As you are removing this code, has the situation for which it was originally put here been handled?

File: sympy/series/gruntz.py [hunk 4]
```diff
@@ -626,9 +625,6 @@ def rewrite(e, Omega, x, wsym):
 
     # Some parts of sympy have difficulty computing series expansions with
     # non-integral exponents. The following heuristic improves the situation:
```
Comment (Sc0rpi0n101) [inline sympy/series/gruntz.py]: The comments here should also be removed if they are not required anymore

**Review by Sc0rpi0n101 — COMMENTED** [review]
It looks like the commit history has a lot of commits that don't even belong to this fix. 
Can you remove these unnecessary commits?

Or if it's getting too messy, I'd suggest you to make a new branch from master, make the changes to the new branch and open a new PR for this fix.

**Comment by Saanidhyavats** [comment]
@smichr I am unable to squash the commits into 1 commit. I tried using `git rebase -i HEAD~`
but didnt get the expected result. Can you guide me on how to squash all the commits?

**Comment by namannimmo10** [comment]
@Saanidhyavats 
```
$ git log    # to check for all the commits
$ git rebase -i <parent commit id>
# It opens the editor showing commits; apart from the first commit replace
# pick to s for other commits. It again opens the editor, put the new commit message
$ git push -f     # to push the commit.
```

**Comment by Saanidhyavats** [comment]
I am closing this issue and will make a new pr

**Pull request closed without merging**